nnpdf_data/nnpdf_data/new_commondata/ATHENA_NC_105GEV_EP/filter.py

```diff
@@ -1,34 +1,11 @@
 from pathlib import Path
-from typing import Optional, Union
 
 import numpy as np
-import pandas as pd
 import yaml
 
-np.random.seed(1234567890)
-
-
-def read_excel(path_xlsx: Path, beams: tuple) -> pd.DataFrame:
-    """Parse the xlsx file containing all the information regarding
-    the projections and returns the ones corresponding to the chosen
-    beam energies.
+from nnpdf_data.athena_utils import read_excel, fluctuate_data, write_data
```

Comment:
Thanks again for this! This looks good to me. My only comment is to move the utils into a dedicated folder as you suggested.
```suggestion
from nnpdf_data.filter_utils.athena_utils import read_excel, fluctuate_data, write_data
```
And for the rest below.